Locals.lua

```diff
@@ -0,0 +1,23 @@
+local getLocals = "getLocals" -- fix bug
+
+function Locals.getLocals(this)
```

Comment:
This is an alias for rawset, effectively

Reply:
yes